documentation/architecture/decisions/0005-giving-users-prometheus-access

```diff
@@ -0,0 +1,24 @@
+# 4. Give users access to Prometheus and Alertmanager
+
+Date: 2018-08-08
+
+## Status
+
+Pending.
+
+## Context
+
+Our users need an easier way to write alerts, they currently have no
+easy way to test queries before writing alerts. Our Prometheus
+service requires authentication which locks our user out of it.
+
+## Decision
+
+We will give our users access to Prometheus so they can use its
+expression browser to test queries when writing alerts. We will
+do this by using IP Whitelisting instead of authentication and
+only allowing our office IP addresses.
```

Comment:
can this ADR capture alternatives to IP whitelisting that we could have chosen, and why we are going with IP whitelisting now, and what might change in future that would cause us to revisit this decision?

Reply:
You have thought harder about this, could you tell me what might cause us to revisit the decision? I'm not sure what to write here

Reply:
I tried to address some of @philandstuff comments and help elaborate the context as requested by @MuriloDalRi